### DISCUSSION TITLE
Any plan to support rdp for remote debugging via rdp

### DISCUSSION THREAD

**Discussion by gengjiawen** [community/community#26877] [Actions]
<p>Like appveyor <a href="https://www.appveyor.com/docs/how-to/rdp-to-build-worker/" rel="nofollow noopener">https://www.appveyor.com/docs/how-to/rdp-to-build-worker/</a>.</p>
<p>Very conveninent feature. It will be much helpful with this feature.</p>

**Comment by BelHFL** [comment]
<p>Hi <a class="mention" href="/gengjiawen">@gengjiawen</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by nelsonjchen** [comment, marked as answer]
<p>For anyone stumbling on this, I came up with a way to RDP into a GitHub Actions build: <a href="https://github.com/nelsonjchen/reverse-rdp-windows-github-actions" rel="noopener nofollow ugc">https://github.com/nelsonjchen/reverse-rdp-windows-github-actions</a></p>

**Comment by gengjiawen** [comment]
<p>Thanks a lot ! this is really has been great help.</p>

**Comment by vikramre1989** [comment]
FastRDP is a tool for brute force attacks on the Remote Desktop Protocol (RDP) on the Windows operating system, which allows users to remotely connect to computers and servers. This tool is written in C# programming language.

I tested this tool, and it works great! I’ll put the link to this tool below. I strongly recommend trying out this tool.

[FastRDP](https://github.com/rensii-1996/RDP-Brute-Force)
